### PR TITLE
Apply GCI format to import order

### PR DESCRIPTION
**Pull request by howardjohn**
Context https://github.com/istio/istio/pull/26130

Pre format this before we introduce the new formatter as required.

This introduces a new stricter ordering on imports. The goal is to avoid
extra diffs in PRs and merge conflicts

T&R agreed plan is to merge this, then merge https://github.com/istio/istio/pull/26130 into common files. This will hopefully avoid any period of breakage between the linter job being updated

cc @daixiang0

### CODE REVIEW

**Review by liamawhite — APPROVED** [review]

**Review by bianpengyuan — APPROVED** [review]

File: galley/pkg/config/source/kube/interfaces.go [hunk 1]
```diff
@@ -18,7 +18,9 @@ import (
 	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp" // import GKE cluster authentication plugin
+
```
Comment (hzxuzhonghu) [inline galley/pkg/config/source/kube/interfaces.go:21]: This is not right

Comment (howardjohn) [reply galley/pkg/config/source/kube/interfaces.go:21]: That is true, looks like it has an extra space. But as long as the tool is consistent may not be a big issue?

**Review by hzxuzhonghu — APPROVED** [review]

File: galley/pkg/config/source/kube/interfaces.go (hunk 1 above)
Comment (daixiang0) [reply galley/pkg/config/source/kube/interfaces.go:21]: I will fix it :)

**Review by morvencao — APPROVED** [review]

**Review by brian-avery — APPROVED** [review]

**Comment by howardjohn** [comment]
@istio/wg-security-maintainers @linsun can you take a look?

**Comment by howardjohn** [comment]
@istio/wg-security-maintainers @linsun can you take a look?

**Review by yangminzhu — APPROVED** [review]

**Review by linsun — APPROVED** [review]

**Pull request merged**